coral/src/domain/topic/topic-api.ts

```diff
@@ -37,4 +37,31 @@ const getTopics = async ({
     .then(transformTopicApiResponse);
 };
 
-export { getTopics };
+const getTopicNames = async ({
```

Comment:
Personal preference, so no need to change. However, I would find this easier to read. I like to extract types to order to improve the readability. 
```
type getTopicNamesArgs = {
  isMyTeamTopics?: boolean;
};

const getTopicNames = ({ isMyTeamTopics = false }: getTopicNamesArgs) => {
  const params = { isMyTeamTopics: isMyTeamTopics.toString() };

  return api.get<KlawApiResponse<"topicsGetOnly">>(
    `/getTopicsOnly?${new URLSearchParams(params)}`
  );
};
```

Reply:
It's also my preference, I just forget it sometimes haha! Thank you ^^

Btw, talking about preferences, I like `interface` more than `type` when `type` is not required. Theres no strong feelings about that by anyone you may be aware of? ^^

Reply:
I've a medium feeling to prefer types and only use interfaces when specific features of interfaces are needed. I feel the whole discussion about that is very opinionated and you can find recs or pro/cons for both approaches equally. So I don't really read into it if someone uses interface or type in a way where I think "oh, this has a special meaning I need to pay attention to." ^^

Reply:
The traditional "gotcha" argument in favor of interface is that it is faster to compile. I think it is really up to preference if you don't need specific features of one of them. I like interfaces because they look less like variable assignments to me, and I like curly braces. I don't think we should enforce one or the other when it doesn;t matter haha